__tests__/app.test.js

```diff
@@ -36,4 +36,42 @@ describe("GET /api" ,() => {
     expect(res.body.endpoints).toEqual(JSON.parse(jsonFileData))
     })
     })
+})
+
+describe("GET /api/articles/:article_id", () => {
+    test("status: 200, responds with an article of a specific ID from the GET request", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toMatchObject( {
```

Comment:
This is nice, you could have also copied across the article from the data folder as an alternative.